bundle-workflow/src/test_workflow/test_results.py

```diff
@@ -0,0 +1,36 @@
+import os
+
+from aws.s3_bucket import S3Bucket
+
+
+class TestResults:
+    def __init__(self, bundle_manifest, test_recorder):
+        self.bundle_manifest = bundle_manifest
+        self.test_recorder = test_recorder
+
+    def to_s3(self, bucket):
+        """
+            Publishes tests results to S3 pulling information from {self.test_recorder}
+            And cleans up all local storage after publishing ({self.test_recorder}.clean_up())
```

Comment:
Fix this comment as TestRecorder cleans it up by itself and TestPublisher doesn't have to be worried about it.

Reply:
TestRecorder is not cleaning up. I removed the clean_up block but it is creating the temporary directory which I assume will be cleaned up eventually by the system?

Reply:
Responded on the other PR so that its clear: https://github.com/opensearch-project/opensearch-build/pull/467/files#r708746645
I'll update the comment on #207